config/areas/site/requests.php

```diff
@@ -1,9 +1,19 @@
 <?php
 
+use Kirby\Panel\Controller\Request\FileItemsRequestController;
+use Kirby\Panel\Controller\Request\PageItemsRequestController;
 use Kirby\Panel\Controller\Request\PageTreeParentsRequestController;
 use Kirby\Panel\Controller\Request\PageTreeRequestController;
 
 return [
+	'items.files' => [
+		'pattern' => 'items/files',
+		'action'  => FileItemsRequestController::class
+	],
+	'items.pages' => [
+		'pattern' => 'items/pages',
+		'action'  => PageItemsRequestController::class
+	],
```

Comment:
`Discussion:` Not sure if these are the best names for those request endpoints. Happy for other suggestions.

Reply:
I thought about it and I can't think of a much better name to be honest. "Preview" would be too close to our page previews. Item is nice and close to what we do in the front and backend. I'm fine with it.